content/en/synthetics/troubleshooting/_index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Synthetics Troubleshooting
+kind: documentation
+description: Troubleshoot common Synthetics isses.
+beta: true
+further_reading:
+- link: "synthetics/"
+  tag: "Documentation"
+  text: "Manage your checks"
+- link: "synthetics/browser_tests"
+  tag: "Documentation"
+  text: "Configure a Browser Test"
+- link: "synthetics/api_tests"
+  tag: "Documentation"
+  text: "Configure an API Test"
+---
+
+If you experience issues setting up or configuring Datadog Synthetics, use this information to begin troubleshooting. If you continue to have trouble, [contact our awesome support team][1].
+
+## Failed requests on a working endpoint
```

Comment:
Let's create a first category called API tests, and then put that paragraph in a subcategory called something like `Set a user-agent`. This way some users might not even need to go through the whole paragraph and will be able to directly understand what they need to do by reading the title!